### PR TITLE
DBCON-234: updates source operation regarding bug-fix

### PR DESCRIPTION
**Pull request by julodiaz**

### CODE REVIEW

File: db/1.8/modules/ROOT/pages/database-documentation.adoc [hunk 1]
```diff
@@ -825,6 +825,7 @@ Updates data in a database.
 
 This operation selects from a table at a regular interval and generates one message per obtained row.  Optionally, you can provide watermark and ID columns. If a watermark column is provided, the values taken from that column are used to filter the contents of the next poll, so that only rows with a greater watermark value are returned. If an ID column is provided, this component automatically verifies that the same row is not picked twice by concurrent polls.
 
+This operation does not support streaming, meaning that access to the result will be done without any additional transformations to the payload. This behavior is identical to the one present in the newly released *Query Single* operation (starting on version 1.9.0).
```
Comment (julodiaz) [inline db/1.8/modules/ROOT/pages/database-documentation.adoc]: @kevintroller it is possible to link on both files the reference to the Query Single operation to the page of the operation on version 1.9??

**Comment by julodiaz** [comment]
This can be merged.

File: db/1.8/modules/ROOT/pages/database-documentation.adoc [hunk 2]
```diff
@@ -825,6 +825,7 @@ Updates data in a database.
 
 This operation selects from a table at a regular interval and generates one message per obtained row.  Optionally, you can provide watermark and ID columns. If a watermark column is provided, the values taken from that column are used to filter the contents of the next poll, so that only rows with a greater watermark value are returned. If an ID column is provided, this component automatically verifies that the same row is not picked twice by concurrent polls.
 
+This operation does not support streaming, and indicates that the access to the results are done without any additional transformations to the payload. This behavior is identical to the the *Query Single* operation released on version 1.9.
```
Comment (dhirsch1) [inline db/1.8/modules/ROOT/pages/database-documentation.adoc]: How does the operation indicate that the access to the results are done without any additional transformations to the payload.?

Comment (dhirsch1) [inline db/1.8/modules/ROOT/pages/database-documentation.adoc]: Do you have to say what version the Query Single operation is in?  If yes, change "released on" to "released in."

**Review by dhirsch1 — DISMISSED** [review]

File: db/1.8/modules/ROOT/pages/database-documentation.adoc [hunk 3]
```diff
@@ -825,6 +825,7 @@ Updates data in a database.
 
 This operation selects from a table at a regular interval and generates one message per obtained row.  Optionally, you can provide watermark and ID columns. If a watermark column is provided, the values taken from that column are used to filter the contents of the next poll, so that only rows with a greater watermark value are returned. If an ID column is provided, this component automatically verifies that the same row is not picked twice by concurrent polls.
 
+his operation does not support streaming, meaning that there is no need to perform additional transformations to the payload in order to access the operation results. This behavior is identical to the the *Query Single* operation released in version 1.9.
```
Comment (dhirsch1) [inline db/1.8/modules/ROOT/pages/database-documentation.adoc]: his  > This

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**